builders/interoperability/xcm/core-concepts/weights-fees.md

```diff
@@ -256,38 +256,55 @@ The total cost is `{{ networks.moonbeam.xcm.transfer_glmr.glmr_cost }} GLMR` for
 
 ### Fee Calculation for External Assets {: #fee-calc-external-assets }
 
-Considering the scenario of Alice sending DOT to Alith's account on Moonbeam, the fees are taken from the amount of xcDOT Alith receives. To determine how much to charge, Moonbeam uses a concept called `UnitsPerSecond`, which refers to the units of tokens that the network charges per second of XCM execution time (considering decimals). This concept is used by Moonbeam (and maybe other parachains) to determine how much to charge for XCM execution using a different asset than its reserve.
+Moonbeam charges fees for external assets based on the weight of the call. Weight is a struct that contains two fields, `refTime` and `proofSize`. `refTime` refers to the amount of computational time that can be used for execution. `proofSize` refers to the amount of storage in bytes that can be used. Since both `refTime` and `proofSize` are integral components of determining a weight, it is impossible to obtain an accurate weight value with just one of these values. Previously, Moonbeam relied on a single weight derived primarily from `refTime` but weight is now defined as a struct taking into account both `refTime` and `proofSize`. 
```

Comment:
proof size is not the amount of storage that can be used. It is related to the the size of the Proof of Validity that this call generates so that polkadot can validate it.

I would remove that "Moonbeam relied on a single weight" as that statement is not future-proof